libass/ass.c

```diff
@@ -232,24 +232,10 @@ static long long string2timecode(ASS_Library *library, char *p)
         ass_msg(library, MSGL_WARN, "Bad timestamp");
         return 0;
     }
-    tm = ((h * 60LL + m) * 60 + s) * 1000 + ms * 10;
+    tm = ((h * 60LL + m) * 60 + s) * 1000 + ms * 10LL;
```

Comment:
Note that this does not fix the `long long` overflow on assignment if the value exceeds `LLONG_MAX` or `LLONG_MIN`. As I noted in the pull request comment, we need to decide what to do in that case.